lib/routes/account.js

```diff
@@ -2459,7 +2459,8 @@ module.exports = (
             .then(
               function () {
                 // Notify the devices that the account has changed.
-                push.notifyPasswordReset(accountResetToken.uid, devicesToNotify)
+                var uid = Buffer(accountResetToken.uid, 'hex')
```

Comment:
Pretty sure: `Error: The first param –uid– of notifyPasswordReset must be a Buffer!`